homeassistant/components/zha/number.py

```diff
@@ -947,3 +948,22 @@ class AqaraThermostatAwayTemp(ZHANumberConfigurationEntity):
     _attr_mode: NumberMode = NumberMode.SLIDER
     _attr_native_unit_of_measurement: str = UnitOfTemperature.CELSIUS
     _attr_icon: str = ICONS[0]
+
+
+@CONFIG_DIAGNOSTIC_MATCH(cluster_handler_names=CLUSTER_HANDLER_THERMOSTAT)
+# pylint: disable-next=hass-invalid-inheritance # needs fixing
+class ThermostatLocalTempCalibration(ZHANumberConfigurationEntity):
+    """Local temperature calibration."""
+
+    _unique_id_suffix = "local_temperature_calibration"
+    _attr_entity_category = EntityCategory.CONFIG
```

Comment:
this is inherited. No need to redefine it

Reply:
"_attr_entity_category", right?

Reply:
yes

Reply:
no problem
https://github.com/home-assistant/core/pull/105765/commits/b8b2e14443c47d86abb2d79c292a330d0233a4f1